client/src/templates/Challenges/projects/SolutionForm.tsx

```diff
@@ -1,5 +1,4 @@
 import React, { Component } from 'react';
-import PropTypes from 'prop-types';
 import { withTranslation } from 'react-i18next';
```

Comment:
```suggestion
import { withTranslation } from 'react-i18next';
import type { WithTranslation } from 'react-i18next';
```
We'll need this lower down.